suite-native/module-home/src/components/BiometricsBottomSheet.tsx

```diff
@@ -0,0 +1,174 @@
+    const [isVisible, setIsVisible] = useState(false);
+    const [biometricsTypes, setBiometricsTypes] = useState<AuthenticationType[]>([]);
+
+    const isFace = biometricsTypes?.filter(biometricsType =>
+        [AuthenticationType.FACIAL_RECOGNITION, AuthenticationType.IRIS].includes(biometricsType),
+    );
+
+    const iconName = isFace ? 'faceId' : 'fingerprint';
+
+    const title = useMemo(() => {
+        if (biometricsTypes?.length === 0) {
+            return <Translation id="moduleHome.biometricsModal.title.unknown" />;
+        }
+        if (isFace) {
+            return <Translation id="moduleHome.biometricsModal.title.faceId" />;
+        }
+        if (Platform.OS === 'android') {
+            return <Translation id="moduleHome.biometricsModal.title.fingerprint" />;
+        }
+        return <Translation id="moduleHome.biometricsModal.title.touchId" />;
+    }, [biometricsTypes, isFace]);
+
+    useEffect(() => {
+        const getSupportedTypes = async () => {
+            const biometricsTypesAvailable = await supportedAuthenticationTypesAsync();
+            setBiometricsTypes(biometricsTypesAvailable);
+        };
+        getSupportedTypes();
+    }, []);
+
+    useEffect(() => {
+        let isMounted = true;
+        let timerId: ReturnType<typeof setTimeout>;
+        const checkBiometrics = async () => {
+            const isBiometricsAvailable = await getIsBiometricsFeatureAvailable();
+            if (isBiometricsAvailable && !isBiometricsOptionEnabled) {
+                timerId = setTimeout(() => {
+                    if (isMounted) {
+                        setIsVisible(true);
+                    }
+                }, SHOW_TIMEOUT);
+            }
+        };
+        checkBiometrics();
+
+        return () => {
+            clearTimeout(timerId);
+            isMounted = false;
+        };
+    }, [isBiometricsOptionEnabled, hasDecidedBiometrics]);
+
+    const handleClose = () => {
+        analytics.report({
+            type: EventType.ModalBiometricsDecision,
+            payload: { enabled: false },
+        });
+        setIsVisible(false);
+        setHasDecidedBiometrics(true);
+    };
+
+    const handleEnable = async () => {
+        const result = await authenticate();
+        if (result?.success) {
+            showToast({
+                variant: 'success',
+                icon: 'check',
+                message: translate('moduleHome.biometricsModal.resultMsg.success'),
+            });
+            analytics.report({
+                type: EventType.ModalBiometricsDecision,
+                payload: { enabled: true },
+            });
+            setIsVisible(false);
+            setHasDecidedBiometrics(true);
+            return;
+        }
+        showToast({
+            variant: 'error',
+            icon: 'warningTriangle',
+            message: translate('moduleHome.biometricsModal.resultMsg.error'),
+        });
+    };
+
+    if (hasDecidedBiometrics) {
+        return null;
+    }
```

Comment:
```suggestion
    if (hasDecidedBiometrics) return null
```

You can check here if you'd like https://www.notion.so/satoshilabs/Basic-syntax-49331c82e0d14fb7a1229b6460009e64

But this is very low prio, just so you know that one liner here would be fine

Reply:
Thanks. I know this is also considered good. But I prefer this over one liner :)